### PR TITLE
Ana/fix handle algos polkadot txs extension

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Needs luniehq/cosmos-keys#89 to work

Fixes polkadot transactions in extension. Concretely these two errors:

image

AND

`createType(MultiSignature):: Unable to create Enum via index 104, in Ed25519, Sr25519, Ecdsa`

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f240a0b75724f06603860e6--lunieio.netlify.app

File: app/src/components/transactions/TransactionIcon.vue [hunk 1]
```diff
@@ -13,9 +14,14 @@ export default {
       required: true,
     },
   },
+  data: () => ({
+    isExtension: config.isExtension,
+  }),
   computed: {
     icon() {
-      return `/img/icons/activity/${this.transactionType}.svg`
+      return this.isExtension
```
Comment (faboweb) [inline app/src/components/transactions/TransactionIcon.vue:22]: This should be handled by webpack. The paths should be the same.

Comment (Bitcoinera) [reply app/src/components/transactions/TransactionIcon.vue:22]: yes, I know, the images don't display :thinking:  but I was seeing this ugly `ERR FILE NOT FOUND` and I thought, let's fix this

Comment (Bitcoinera) [reply app/src/components/transactions/TransactionIcon.vue:22]: ok, so then I will simply create an `img/icons` folder in extension and that should fix it too. No need to use webpack for that then

Comment (faboweb) [reply app/src/components/transactions/TransactionIcon.vue:22]: I mean the images shouldn't be in our repo twice. Webpack should copy them in the right place

Comment (Bitcoinera) [reply app/src/components/transactions/TransactionIcon.vue:22]: O.o do yo need to rebuild the whole thing when you change the `webpack.config`? I think you do :sweat:

Comment (faboweb) [reply app/src/components/transactions/TransactionIcon.vue]: yep

**Comment by Bitcoinera** [comment]
It is not working... getting the icons from the `app` workspace I mean. Can you please help @faboweb ? 

I already tried resolving `img` and `/img`... Thank you :pray:

File: extension/src/messageHandlers.js [hunk 2]
```diff
@@ -118,7 +122,7 @@ export async function signMessageHandler(
 export async function walletMessageHandler(message, sender, sendResponse) {
   switch (message.type) {
     case 'GET_WALLETS': {
-      sendResponse(getWalletIndex())
+      sendResponse(getWalletIndex(true))
```
Comment (faboweb) [inline extension/src/messageHandlers.js:125]: why do you need to pass `true` here?

Comment (Bitcoinera) [reply extension/src/messageHandlers.js:125]: to get the enriched version, with network and crypto in

Comment (faboweb) [reply extension/src/messageHandlers.js:125]: why isn't that the default?

Comment (Bitcoinera) [reply extension/src/messageHandlers.js:125]: :shrug:

**Comment by Bitcoinera** [comment]
I retested with the new cosmos-keys. This is working great :+1: 

Let's merge!

PS: OK, the icons are still not there... Still ERR FILE NOT FOUND xD but minor thing right?

**Comment by Bitcoinera** [comment]
Merge! :woman_dancing:

**Comment by faboweb** [comment]
I still get:

```
Failed to load resource: net::ERR_FILE_NOT_FOUND
popup.html#/approve:1 Unchecked runtime.lastError: The message port closed before a response was received.
popup.js:4647 Uncaught (in promise) Error: Redirected from "/approve" to "/success" via a navigation guard.
    at ut (popup.js:4647)
    at st (popup.js:4647)
    at popup.js:4647
    at popup.js:4647
    at c (popup.js:2188)
    at Generator._invoke (popup.js:2188)
    at Generator.forEach.e.<computed> [as next] (popup.js:2188)
    at r (popup.js:18)
    at s (popup.js:18)
Staked.svg:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
```

**Pull request merged**